### ISSUE TITLE
Create .intunewin file: WARNING: Unexpected error occurred while packaging Win32 app. Return code from invocation: -532462766 

### ISSUE THREAD

**Issue by subsense33** [MSEndpointMgr/IntuneWin32App#139]
When attempting to follow the instructions provided in the GitHub repository

[https://github.com/MSEndpointMgr/IntuneWin32App?tab=readme-ov-file#package-application-source-files-into-win32-app-package-intunewin](url)

 For packaging application source files into a Win32 app package using New-IntuneWin32AppPackage, I consistently encounter the following error:

WARNING: Unexpected error occurred while packaging Win32 app. Return code from invocation: -532462766
I'm using the exact code as outlined in the readme.md:

```
# Package MSI as .intunewin file
$SourceFolder = "C:\Win32Apps\Source\7-Zip"
$SetupFile = "7z1900-x64.msi"
$OutputFolder = "C:\Win32Apps\Output"
New-IntuneWin32AppPackage -SourceFolder $SourceFolder -SetupFile $SetupFile -OutputFolder $OutputFolder -Verbose
```
If you have any insights into resolving this issue or if there are additional steps to consider, I would greatly appreciate your assistance.

Tested on multiple devices.

![error](https://github.com/MSEndpointMgr/IntuneWin32App/assets/156912460/c061e812-6b34-4b7f-9bfc-46409b9240ff)

**Comment by subsense33** [comment]
Tested several version of IntuneWinAppUtil.exe
They all produce the same error.
No issues running IntuneWinAppUtil.exe standalone

**Comment by soldier9945** [comment]
Hello,

I just researched this for the 3rd time as I have automation that uploads apps in a script. The last 2 times I had to change code in the module itself or some people here had to change to an older IntuneWInAppUtil.exe version as you have tried.

I have just upgraded to the Module to version 1.4.3 and specified in my script's first line:

`#requires -Modules @{ ModuleName="IntuneWin32App"; RequiredVersion="1.4.3" }`

I am currently executing this script with:
PSVersion                      7.3.10
PSEdition                      Core

I haven't changed anything after upgrading the module. I hope this helps you as well.

**Comment by subsense33** [comment]
Thanks, for now I solved this by directly IntuneWinAppUtil.exe from my PoweShell script. Not as nice as downloaden the most recent version, but is does the job

**Comment by That-Annoying-Guy** [comment]
I think this is related to Issue https://github.com/MSEndpointMgr/IntuneWin32App/issues/134
**New-IntuneWin32AppPackage** uses the module's **Invoke-Executable** function.

Supposedly there will be a fix in 1.4.4 but I see nothing yet.

**Comment by NickolajA** [comment]
1.4.4 is out now, please verify if the issue still persist. Reopen the issue if there's still a problem :)
